src/redux/features/Plantings/plantingsSelectors.ts

```diff
@@ -9,3 +11,29 @@ export const selectPlantingsDateRange = (state: RootState, dateRange: string[])
     }
     return planting.createdTime > dateRange[0] && (dateRange.length < 2 || planting.createdTime < dateRange[1]);
   }) ?? [];
+
+export const selectPlantingSiteReportedPlants = (state: RootState, plantingSiteId: number) =>
+  state.plantingSiteReportedPlants;
```

Comment:
We need to key this off `plantingSiteId`, otherwise we use last retrieved results which may not apply to `plantingSiteId`

Reply:
@constanzauanini can you hold off on this suggestion? wondering if we need one API that returns total reported plants for ALL sites. If so, let's bring this up with Steve?

Reply:
Ok, there's a PR out that enhances the plantings search with totalPlants for subzones.
We can help generate the query if needed.